committing.rst

```diff
@@ -428,10 +428,9 @@ Prefix the backport pull request with the branch, for example::
    
 Note that cherry_picker.py_ adds the branch prefix automatically.
    
-Apply the label ``cherry-pick or X.Y`` to the backport pull request.  Once the backport
-pull request has been created, remove the ``needs backport to X.Y`` label from the
-original pull request on ``master``.  Only Core Developers can apply labels to GitHub
-pull requests.  
+Once the backport pull request has been created, remove the
+``needs backport to X.Y`` label from the original pull request on ``master``.
```

Comment:
It's actually possible that the cherry-pick is on master, e.g. if something is fixed in `master` and so initially was patched against `3.6` and then backported to `3.5`.